reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -594,56 +595,56 @@ public Emission getReason() {
 
 		/**
 		 * Try emitting a non-null element, generating an {@link Subscriber#onNext(Object) onNext} signal.
-		 * The result of the attempt is represented as an {@link Emission}, which possibly indicates error cases.
+		 * The result of the attempt is represented as an {@link EmitResult}, which possibly indicates error cases.
 		 * <p>
 		 * Might throw an unchecked exception in case of a fatal error downstream which cannot
 		 * be propagated to any asynchronous handler (aka a bubbling exception).
 		 *
 		 * @param t the value to emit, not null
-		 * @return {@link Emission}
+		 * @return {@link EmitResult}
```

Comment:
These do not provide any added value, users can already discover the return type.

Reply:
improved the comments